java-checks/src/test/files/checks/mismatchPackage/with.dots/PackageWithDots.java

```diff
@@ -0,0 +1,3 @@
+package mismatchPackage.with.dots; // Noncompliant {{This file "$PackageWithDots.java" should be located in "$mismatchPackage/with/dots" directory, not in "$src/test/files/checks/mismatchPackage/with.dots"(Do not use dots in directory names).}}
```

Comment:
This test should fail on windows, I don't see a simple solution to test the message. We could probably accept that the message is not tested.

Reply:
Good point, I'll remove the message then